helix/pages/1_New_Experiment.py

```diff
@@ -110,15 +111,15 @@ def _entrypoint(save_dir: Path):
 st.header("New Experiment")
 st.write(
     """
-    Here you can start a new experiment. Once you create one, you will be able
-    to select it on the Machine Learning & Feature Importance pages.
+    Here you can create a new experiment. Once created, you will be able
+    to select it on the Data Preprocessing, Data Visualisation and Train Models pages.
```

Comment:
```suggestion
    Here you can create a new experiment. Once created, you will be able
    to select it on the Data Preprocessing, Data Visualisation, Train Models,
    Feature Importance & View Experiments pages.
```
Mention all the pages

Reply:
@dcl10 no you cannot do feature importance and view experiments right after creating the experiment. You can only perform feature importance AFTER training the models. The message changes you suggested is a little misleading. Yes one can see the page, but cannot do anything until the models are created. Does that make sense?